server/routes/visit.test.ts

```diff
@@ -159,6 +159,9 @@ describe('GET /visit/:reference', () => {
       modifiedTimestamp: '2022-02-14T10:05:00',
     }
 
+    const fakeDate = new Date('2022-01-01')
+    jest.useFakeTimers({ advanceTimers: true, now: fakeDate })
```

Comment:
I've looked into this a bit further and, instead of my original `advanceTimers` suggestion, you could alternatively do:
```javascript
jest.useFakeTimers({ doNotFake: ['nextTick'], now: fakeDate })
```
This fixes the problem because it's the faking of [`nextTick` ](https://nodejs.org/en/docs/guides/event-loop-timers-and-nexttick/) that causes it to hang. 

`advanceTimers` automatically keeps it running, but we could just not fake it in the first place 🤷‍♂️ ! (I think either is fine...)